lib/PuppeteerSharp.Tests/ScreencastTests/PageScreencastTests.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading.Tasks;
+using NUnit.Framework;
```

Comment:
We also need this :)

```suggestion
using NUnit.Framework;
using PuppeteerSharp.Nunit;
```